### ISSUE TITLE
MacOS Fails to load en model

### ISSUE THREAD

**Issue by whicks1** [aguang-xyz/auto-caption#2]
Right after moviePy reports done I get the following (MacOS 10.15.7)

`Traceback (most recent call last):
  File "/usr/local/bin/auto-caption", line 7, in <module>
    main()
  File "/usr/local/lib/python3.8/site-packages/auto_caption/__init__.py", line 226, in main
    auto_caption(args.video, output, fmt=fmt, lang="en")
  File "/usr/local/lib/python3.8/site-packages/auto_caption/__init__.py", line 199, in auto_caption
    sentences = segment_setences(words)
  File "/usr/local/lib/python3.8/site-packages/auto_caption/__init__.py", line 119, in segment_setences
    for tokens2d in tqdm(nnsplit.NNSplit(lang).split([content])):
  File "backend.py", line 5, in create_session
  File "/usr/local/lib/python3.8/site-packages/onnxruntime/capi/session.py", line 195, in __init__
    self._create_inference_session(providers, provider_options)
  File "/usr/local/lib/python3.8/site-packages/onnxruntime/capi/session.py", line 200, in _create_inference_session
    sess = C.InferenceSession(session_options, self._model_path, True, self._read_config_from_model)
onnxruntime.capi.onnxruntime_pybind11_state.NoSuchFile: [ONNXRuntimeError] : 3 : NO_SUCHFILE : Load model from en failed:Load model en failed. File doesn't exist`

latest vosk,onnxruntime, and nnsplit via pip3. libbomp via homebrew. attempted pip install of kaldi-active-grammar and dragonfly[kalidi] afterwards with no effect. ~/.auto-captions/models/en/... was created from zip download on the first run of the program.  Apologies if I'm missing something obvious, new area for me.

**Comment by aguang-xyz** [comment]
Hi,

Than you for the interest of my project. And sorry for this late reply. Every time this program starts, it will check if the language model has been cached in the `~/.auto-caption` folder. It seems pretty possible that there is some broken cache left there.

So I suggest you to find the protential issue following these steps:

1. Make sure the latest version of `ffmpeg` is properly installed on your system. On OSX, your could run:

```bash
brew upgrade ffmpeg
```

2. Update `auto-caption` to the latest version.

```bash
pip install -U auto-caption
```

3. Clean your local model cache.

```bash
rm -rf ~/.auto-caption
```

4. Try some video.

```bash
auto-caption xxx.mkv
```

Hope this could solve your problem.

Regards,
Guangrui Wang

**Comment by aguang-xyz** [comment]
Hi, I just released v0.0.6 which should solve this issue.

Please upgrade `auto-caption` via `pip3 install -U auto-caption` and have a try.

🤭 Give me a star if it works.

**Comment by aguang-xyz** [comment]
Hi~ Any update?
